### ISSUE TITLE
Simplified launch of environments

### ISSUE THREAD

**Issue by ciyer** [SwissDataScienceCenter/renku-ui#1152]
## Description

With the ability to specify environment settings defaults, there is no need to ask the user so many questions to launch an environment. 

There should be a start/connect button that the user can just click to start an environment with the default settings (no form to fill out). If an environment is already running for the project, then the user can connect to it. There is an "advanced" start option that will let the user fill out the form.

The form should also be simpler. The advanced options can be hidden by default.

## Differences to mock-up

In the mock up, there is no environments tab, but we will make that change later in another issue. Let us start by simplifying launch first.

In the mock up there is a the expectation that start becomes connect if a session is running, but that feature is being blocked by the UI server, so let us leave the button as start always and when clicked, the UI will decide if it should start a new server or connect.

The start form should have a back to project button.

## Mock Ups

![image](https://user-images.githubusercontent.com/1196411/106929331-872fdd80-6714-11eb-89c8-844774fb48f3.png)

**Comment by lorenzo-cavazzi** [comment]
Tip: remember to check for autosaved content from the new `GET /autosaves` endpoint SwissDataScienceCenter/renku-notebooks/issues/584
